modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldDefaultValue.testcase

```diff
@@ -21,12 +21,99 @@ definition {
 		}
 	}
 
-	@description = "LPS-163716 - Verify it's not possible to apply the default value when updating an existing object entry"
+	@description = "LPS-181207 - Verify it's not possible to apply the default value when updating an existing object entry"
 	@priority = 3
 	test CannotSetForExistingObjectEntry {
+		property custom.properties = "feature.flag.LPS-163716=true";
 
-		// TODO pending implementation
+		task ("Given a picklist with several items") {
+			Picklist.addPicklistViaAPI(picklistName = "Custom Picklist 181207");
+
+			for (var itemNumber : list "1,2,3,4,5") {
+				Picklist.addPicklistItemViaAPI(
+					itemKey = ${itemNumber},
+					itemName = "Item Test ${itemNumber}",
+					picklistName = "Custom Picklist 181207");
+			}
+		}
+
+		task ("And given a custom object with a picklist field and an object entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 181207",
+				objectName = "CustomObject181207",
+				pluralLabelName = "Custom Objects 181207");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Picklist",
+				fieldLabelName = "Picklist Field",
+				fieldName = "picklistField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject181207",
+				picklistName = "Custom Picklist 181207");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject181207");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject181207");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject181207");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry Test");
```

Comment:
Can we add the entry using the API here?

Reply:
Replaced by api